python/README.md

```diff
@@ -20,7 +20,7 @@ This package is released under the Apache 2.0 license. See the LICENSE file.
 ## Installation
 
 This package has the following requirements:
- - a recent version of scikit-learn. Version 0.17 has been tested, older versions may work too.
+ - Sklearn version >= 0.18.1
```

Comment:
Sklearn 0.18.0 doesn't have MaskedArray which is needed by GridSearchCV so I don't think it would work.